core/src/main/scala/uk/gov/nationalarchives/tdr/api/core/GraphQlServer.scala

```diff
@@ -58,15 +60,23 @@ object GraphQlServer {
       FileType,
       arguments = List(filePathArg, ConsignmentIdArg),
       resolve = ctx => ctx.ctx.files.create(ctx.arg(filePathArg), ctx.arg(ConsignmentIdArg))
+    ),
+    Field(
+      "updateFileChecksum",
```

Comment:
As discussed in person, there should probably be separate mutations for server-side and client-side checksums.

Reply:
I've split them out into `updateServerSideFileChecksum` and `updateClientSideFileChecksum`